### PR TITLE
Document the Blazor WebEventDescriptor.EventArgsType breaking change

### PR DESCRIPTION
**Pull request by scottaddie**
Document the breaking change described at https://github.com/aspnet/Announcements/issues/453

### CODE REVIEW

File: docs/core/compatibility/aspnet-core/6.0/blazor-eventargstype-property-replaced.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<!--
+
+## Category
+
+ASP.NET Core
+
+## Affected APIs
+
+`Overload:Microsoft.AspNetCore.Components.RenderTree.WebEventDescriptor.EventArgsType`
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/6.0/blazor-eventargstype-property-replaced.md:48]:
```suggestion
`P:Microsoft.AspNetCore.Components.RenderTree.WebEventDescriptor.EventArgsType`
```

File: docs/core/compatibility/aspnet-core/6.0/blazor-eventargstype-property-replaced.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Breaking change: Blazor: WebEventDescriptor.EventArgsType property replaced"
+description: "Learn about the breaking change in ASP.NET Core 6.0 titled Blazor: WebEventDescriptor.EventArgsType property replaced"
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/6.0/blazor-eventargstype-property-replaced.md:3]:
```suggestion
description: "Learn about the breaking change in ASP.NET Core 6.0 where the WebEventDescriptor.EventArgsType property is replaced by the EventName property."
```

**Review by gewarren — APPROVED** [review]

File: docs/core/compatibility/6.0.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Breaking changes in .NET 6.0
 description: Navigate to the breaking changes in .NET 6.0.
-ms.date: 02/16/2021
```
Comment (Youssef1313) [inline docs/core/compatibility/6.0.md:4]: Consider adding no-loc metadata for "Blazor"

File: docs/core/compatibility/aspnet-core/6.0/blazor-eventargstype-property-replaced.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Breaking change: Blazor: WebEventDescriptor.EventArgsType property replaced"
+description: "Learn about the breaking change in ASP.NET Core 6.0 where the WebEventDescriptor.EventArgsType property is replaced by the EventName property."
+author: scottaddie
+ms.author: scaddie
+ms.date: 02/24/2021
+---
```
Comment (Youssef1313) [inline docs/core/compatibility/aspnet-core/6.0/blazor-eventargstype-property-replaced.md:7]: Blazor needs no-loc

**Pull request merged**